src/js/wrappers/InputWrapper.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiQuestionLine } from 'react-icons/ri';
+import { Tooltip } from 'react-tippy';
+
+import Translate from 'utils/Translate';
+
+import './style.scss';
+
+const InputWrapper = ({
+  children,
+  tooltip,
+  required,
+  button,
+  title,
+  errorMessage,
+}) => (
+  <div className="input-wrapper-container">
+    <div>
+      <div className="input-wrapper-title">
+        {title && <Translate id={title?.id} defaultMessage={title?.defaultMessage} />}
+        {tooltip && (
+        <Tooltip
+          html={(
+            <span className="p-1">
+              <Translate id={tooltip.id} defaultMessage={tooltip.defaultMessage} />
+            </span>
+              )}
+        >
+          <RiQuestionLine />
+        </Tooltip>
+        )}
+        {required && <span className="input-wrapper-asterisk">&#42;</span>}
+      </div>
+      {button && (
+      <div
+        onClick={button.function}
```

Comment:
```suggestion
        onClick={button.onClick}
```
let's follow the react naming pattern for callback props that handle onClick functionality